docs/getting-started/setup.mdx

```diff
@@ -1,440 +1,164 @@
+   ### What's happening?
 
-2. Download and update Kubernetes config file
+   We're cloning our Docker Compose [quickstart repo](https://github.com/mage-ai/compose-quickstart.git), copying `dev.env` to `.env`, which is ignored by git in our configuration (to avoid exposing secrets), then running `docker compose up` to start a Mage server.
 
-   * Download Mage Kubernetes config file. https://github.com/mage-ai/mage-ai/blob/master/kube/app.yaml
-   * Replace the `/path/to/mage_project` in the config yaml with the path that you want to use to store your Mage projects.
+   </Accordion>
+   <Accordion title="Docker run" icon="docker">
 
-3. Run Mage app in Kubernetes cluster
+   ### Get started
 
-   * Install command line tool [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) if you haven't.
-   * Run command `kubectl create -f kube/app.yaml` to run Mage in a Kubernetes pod.
-   * You can check the pod status with command `kubectl get pods -o wide`.
-   * Set up port forwarding with command `kubectl port-forward mage-server 6789:6789`.
+   First, be sure [Docker](https://docs.docker.com/engine/install/) is installed. Next, create a new folder for your project, change directory into that folder, and run the following from your terminal:
 
-3. Open http://localhost:6789 in your browser and build a pipeline.
+   <CodeGroup>
+      ```bash Mac/Linux
+      docker run -it -p 6789:6789 -v $(pwd):/home/src mageai/mageai /app/run_app.sh mage start [project_name]
+      ```
+      ```bash Windows Command Line
+      docker run -it -p 6789:6789 -v %cd%:/home/src mageai/mageai /app/run_app.sh mage start [project_name]
+      ```
 
-4. (Optional) Enable blocks to be executed in separate Kubernetes pods.
+      ```bash PowerShell
+      docker run -it -p 6789:6789 -v ${PWD}:/home/src mageai/mageai /app/run_app.sh mage start [project_name]
+      ```
+   </CodeGroup>
 
-   If you want the pipeline blocks to be executed in separate containers, you can follow the [guide](/production/configuring-production-settings/compute-resource#kubernetes-executor) to set `executor_type` of a block to `k8s` in pipeline's `metadata.yaml`.
+   Once the server is running, open `http://localhost:6789` in your browser and explore! We recommend one of our [guides](/guides/overview) to for inspiration.
 
----
+   ### What's happening?
 
-## Download new version of Mage
+   We're executing `docker run`, which runs a Docker container from the Mage image. additionally, we're _mounting a volume_ (the `-v` command) to persist our project files on our container. This mounts the current folder to `home/src` in the mage container. We're executing our docker image `mageai/mageai` and running a script to start up the container.
 
-### Using Docker
+   If you'd like to connect to an external database, either Postgres or MSSQL, using Docker containers, please see the [external database quickstart](/getting-started/external-database-quickstart).
```

Comment:
Where is the doc for "/getting-started/external-database-quickstart"?

Reply:
I removed this reference. This is more of a docker tutorial than a mage quickstart